docs/index.rst

```diff
@@ -17,6 +17,7 @@ The Software Factory documentation is the central space where we collect all use
     chapters/ci-and-cd/index
     chapters/sdk/index
     chapters/sde/index
+    chapters/oso/index
```

Comment:
Not sure about the path name here. OSO is not as obvious as an acronym as SDK for example. But I don't have a strong opinion about it anyway.

Reply:
OSO is the acronym used for the open source office (although so far mostly internally) so I think it makes sense to keep using it.

Reply:
Maybe it's nicer to spell it out? "chapters/opensourceoffice/index" ?